tests/DwitTech.AccountService.Core.Tests/Services/ActivationServiceTests.cs

```diff
@@ -24,13 +34,63 @@ public void SendActivationEmail_Returns_True(string fromMail, string toMail, str
                 .AddInMemoryCollection(inMemorySettings)
                 .Build();
 
-            var actService = new ActivationService(configuration);
+            var options = new DbContextOptionsBuilder<AccountDbContext>()
+                .UseInMemoryDatabase(Guid.NewGuid().ToString())
+                .Options;
+
+            var mockDbContext = new Mock<AccountDbContext>(options);
+            IUserRepository repository = new UserRepository(mockDbContext.Object);
+            var iConfig = new Mock<IConfiguration>();
```

Comment:
This line seems irrelevant since it is never passes into the ActivationService. A configuration variable exists in line 33, and is passed into the ActivationService in line 44, so iConfig is never used. Please remove.

Reply:
> This line seems irrelevant since it is never passes into the ActivationService. A configuration variable exists in line 33, and is passed into the ActivationService in line 44, so iConfig is never used. Please explain its relevance to me if I'm wrong. Otherwise, you should remove it.

Resolved